packages/mobile/src/navigator/types.tsx

```diff
@@ -142,6 +142,9 @@ export type StackParamList = {
     reclaimPaymentInput: EscrowedPayment
   }
   [Screens.RegulatoryTerms]: undefined
+  [Screens.AddressFetchLoading]: {
```

Comment:
My instinct would have been to make this a dialog / modal that can pop up anywhere (like a hidden one on the root App component on something) instead of a screen that we briefly navigate to and then away from. Can you clarify why a screen was used?

Reply:
I think that works too. @nityas @coreycelo do you have a preference for a loading screen or loading modal in this case?

Context is that this will happen every time a user presses "send" on a payment request notification and should last ~1 second.

Reply:
In order of preference:
1. Some form of "inline" spinner on the current screen—in or near the initiating button
2. Fullscreen spinner
3. Modal spinner

The pref for 2 I can see depending on appearance/performance. Ideally, "navigating" to a white screen with a spinner after the Review screen (which itself is on a white bg) won't feel like navigating at all—can we use either no transition or a fade/dissolve? It could feel simply like the Review screen is processing.

Reply:
Changed to inline spinner